sdk/tvOS/OoyalaTVSkinSDK/OOOoyalaTVPlayerViewController.m

```diff
@@ -31,36 +34,52 @@ @interface OOOoyalaTVPlayerViewController ()
 
 @implementation OOOoyalaTVPlayerViewController
 
-- (instancetype)init {
-  if (self = [super init]) {
-    _showsPlaybackControls = NO;
-  }
-  return self;
-}
-
-- (instancetype)initWithPlayer:(OOOoyalaPlayer *)player {
-  if (self = [self init]) {
-    _player = player;
-  }
-  return self;
-}
+#pragma mark lifecyle methods
 
 - (void)viewWillAppear:(BOOL)animated {
   [super viewWillAppear: animated];
 
-  if (self.player) [self setupViewController];
+  [self setupViewController];
+  [self addGestures];
 }
 
-- (void)viewWillDisappear:(BOOL)animated {
+- (void)viewDidDisappear:(BOOL)animated {
+  [super viewDidDisappear:animated];
+  [self removeGestures];
+}
+
+- (void)dealloc {
   [self removeObservers];
-  [super viewWillDisappear:animated];
+}
+
+#pragma mark private helper functions
+- (void)addGestures {
+  self.tapForwardGesture = [[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(seek:)];
```

Comment:
Any chance you can put all gesture/button related interactions into a separate file?  Anything involving responding to end-user interactions.

perhaps move addGestures and removeGestures, with their selectors, into a separate class.